### ISSUE TITLE
Page: Book a demo - Review

### ISSUE THREAD

**Issue by Yndira-FlowForge** [flowforge/website#669]

**Comment by ZJvandeWeg** [comment]
With @zackwasli I discussed getting a few more details at the time of booking. We should better understand what the use case is for the demo, so we can adapt preemptively

**Comment by Yndira-FlowForge** [comment]
That would mean adding a field to the form, right? Or did you have something else in mind?

**Comment by zackwasli** [comment]
Yes a "Notes" field would be great. @ZJvandeWeg should we just have a Notes field, or should we title it with something like: "How can we help you?"

**Comment by joepavitt** [comment]
out _"is there anything in particular you'd like to see or learn about?"_ - given that it will be a text area, rather than single line entry, have the space for a more verbose placeholder

**Comment by ZJvandeWeg** [comment]
The notes part is added, but we should review the flow of the page. The form feels cramped filling it out, and I hope there's a iteration to fix this?
